build.gradle

```diff
@@ -99,7 +99,7 @@ task integration(type: Test) {
   group = "Verification"
   testClassesDirs = sourceSets.integrationTest.output.classesDirs
   classpath = sourceSets.integrationTest.runtimeClasspath
-  failFast = true
+  failFast = false
```

Comment:
Is this changed intended / desirable?

Reply:
I asked Marc to change this. failFast stops any more tests from running once 1 has failed. So you you don't get the full picture of how many failing tests there are. i didn't know why we would want that.